_posts/2017-06-27-Increasing-Rusts-Reach.md

```diff
@@ -0,0 +1,310 @@
+---
+layout: post
+title: "Increasing Rust’s Reach"
+author: Carol Nichols
+---
+
+One of [Rust's 2017 goals](https://blog.rust-lang.org/2017/02/06/roadmap.html) is to make it easier
+to become productive in Rust by reducing its learning curve. We want Rust to be accessible to as
+many people as possible because Rust is an enabling technology that can unlock systems programming
+for more people. But there’s a bit of a bootstrapping problem here: if we want to reach new people,
+we can’t do it by relying solely on the skills and perspectives of our existing community. So we’ve
+decided to do an experiment, and we need your help!
+
+We’re looking for people inside and outside Rust’s current community from groups and backgrounds
+that are underrepresented in the Rust world. We want to partner with you to make Rust a more
+inclusive, approachable and impactful project, while supporting your success on personal goals.
+
+We have a team of Rust community leaders to pair you with. This group isn’t particularly diverse;
+this is where the Rust community is right now. We acknowledge that we have lots of work to do, and
+this initiative is part of that work. We’re all committed to improving the diversity of the Rust
+community and including more kinds of people.
+
+Each Rust team leader has proposed a project in a variety of areas, such as improving Rust itself,
+working on Rust web tools, and improving usability. We’re looking for a variety of experiences and
+skill sets! We’re also open to project ideas of your own.
+
+We’re asking for a time investment of 3-5 hours per week between Aug 7 and Nov 6 (or a shorter time
+period within those dates). The exact scope of each of the projects is flexible, depending on your
+availability and goals. We expect you to check in at least weekly with a summary of your progress,
+and we expect you to produce a deliverable by the end of your time working on the project. The
+deliverable will vary based on the project; it might be pull requests, recommendations,
+documentation, or tutorials, for example.
+
+As thanks for your participation, Mozilla will cover flight, hotel, and ticket costs to the Rust
+conference of your choice out of:
+
+- RustConf, Aug 18-19 in Portland, OR, USA
+- RustFest, Sept 30-Oct 1 in Zurich, Switzerland
+- Rust Belt Rust, Oct 26-27 in Columbus, OH, USA
+
+Your pair will give you the context and tools needed to help you make an impact on an important
+area of Rust. You’ll also have access to a private Slack to chat with the other participants and
+Rust team members involved in this initiative. We’re planning on highlighting the outcomes of this
+experiment and recognizing your contributions explicitly; we value these projects and your
+contributions to them highly!
+
+The groups that are underrepresented in technology and in the Rust community that we want insights
```

Comment:
How about: "Some groups ... that we would especially love insights from"? Just to make sure people who don't exactly fit on this list still feel open to applying?